Orange/widgets/evaluate/owparameterfitter.py

```diff
@@ -0,0 +1,634 @@
+from typing import Optional, Callable, Collection, Sequence
+
+import numpy as np
+from AnyQt.QtCore import QPointF, Qt, QSize
+from AnyQt.QtGui import QStandardItemModel, QStandardItem, \
+    QPainter, QFontMetrics
+from AnyQt.QtWidgets import QGraphicsSceneHelpEvent, QToolTip, \
+    QGridLayout, QSizePolicy, QWidget
+
+import pyqtgraph as pg
+
+from orangewidget.utils.itemmodels import signal_blocking
+from orangewidget.utils.visual_settings_dlg import VisualSettingsDialog, \
+    KeyType, ValueType
+
+from Orange.base import Learner
+from Orange.data import Table
+from Orange.evaluation import CrossValidation, TestOnTrainingData, Results
+from Orange.evaluation.scoring import Score, AUC, R2
+from Orange.modelling import Fitter
+from Orange.util import dummy_callback
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.utils import userinput
+from Orange.widgets.utils.concurrent import ConcurrentWidgetMixin, TaskState
+from Orange.widgets.utils.multi_target import check_multiple_targets_input
+from Orange.widgets.utils.widgetpreview import WidgetPreview
+from Orange.widgets.visualize.owscatterplotgraph import LegendItem
+from Orange.widgets.visualize.utils.customizableplot import \
+    CommonParameterSetter, Updater
+from Orange.widgets.visualize.utils.plotutils import PlotWidget, \
+    HelpEventDelegate
+from Orange.widgets.widget import OWWidget, Input, Msg
+
+N_FOLD = 7
+MIN_MAX_SPIN = 100000
+ScoreType = tuple[int, tuple[float, float]]
+# scores, score name, tick label
+FitterResults = tuple[list[ScoreType], str, str]
+
+
+def _validate(
+        data: Table,
+        learner: Learner,
+        scorer: type[Score]
+) -> tuple[float, float]:
+    # dummy call - Validation would silence the exceptions
+    learner(data)
```

Comment:
I don't like this.
A better solution would be to change the evaluation classes below so that they raise the learner's error instead of silently suppressing them (at least optionally, controlled with an argument?)